open-source-deps.md

```diff
@@ -1,9 +1,9 @@
 Open-Source
 ===========
 
-Kubecost has an open core business model. 
+Kubecost has open source at the center of our business model. 
 
-Our commercial project is based on our cost-model open-source project, which has an Apache 2.0 license. 
-Here is a list of the direct [open-source dependencies](https://github.com/kubecost/cost-model/blob/master/go.mod#L9) that we also include as part of our project.
+Our commercial project is built on top of the [OpenCost CNCF Sandbox project](https://www.cncf.io/projects/opencost/), which has an Apache 2.0 license. 
```

Comment:
Commercial products or product instead of project?

Reply:
Yep!